### ISSUE TITLE
else: v sortable modulu se špatně formátuje

### ISSUE THREAD

**Issue by horacekj** [fi-ksi/web-frontend#286]

**Comment by horacekj** [comment]
Nelze rozumně vyřešit.

**Comment by effa** [comment]
Špatné odsazování je problém, zvlášť když je kód jen trochu složitější, viz například https://ksi.fi.muni.cz/ulohy/103. Pokud to nejde/nezvládneme vyřešit, pak je podle mě nutné zakázat "else" používat.

Trochu jsem nad tím přemýšlel a napadla mě dvě možná řešení, ale možná jsou příliš ad-hoc nebo obtížně implementovatelná, to nevím:

1) Pokud nechci sortable použít na pseudokód, ale na Python, musí mít každý else párující ukončení pomocí komentáře `# end of else`, podobně pro cykly.

2) Řešitel si musí (a může) odsazování určovat sám (blok se přichycuje k svislé čáře určující odsazení). Tak to mají zde: http://interactivepython.org/runestone/static/overview/overview.html#self-check-questions (potřeba sjet kousek dolů...)

**Comment by horacekj** [comment]
Souhlasím, že je to problém. V současnosti bohužel nemám technické kapacity na to to řešit.

**Comment by jirkadanek** [comment]
Jee, to zni jako https://en.wikipedia.org/wiki/Dangling_else, klasika ;)

Pokud je to tohle, tak napr. Scratch to myslim resi hezky, https://scratch.mit.edu/projects/editor/?tutorial=home. If-then a If-then-else je jeden nedelitelny blok se dvema nebo tremi sloty, na podminku, then vetev (a else vetev). Bylo by hezke kdyby to slo odnekud "ukrast".

![image](https://user-images.githubusercontent.com/17877663/77471307-556e9a80-6e12-11ea-9e49-136437930158.png)
